src/core_plugins/kibana/public/visualize/listing/no_visualizations_prompt.js

```diff
@@ -26,7 +27,7 @@ import {
   KuiButtonIcon,
 } from '@kbn/ui-framework/components';
 
-export function NoVisualizationsPrompt() {
+function NoVisualizationsPromptComponent({ intl }) {
```

Comment:
NoVisualizationsPromptComponent => NoVisualizationsPromptUi
we have such agreement